gst/tensor_transform/tensor_transform.c

```diff
@@ -191,9 +191,32 @@ gst_tensor_transform_init (GstTensor_Transform * filter)
 static const gchar *gst_tensor_transform_mode_string[] = {
   [GTT_DIMCHG] = "dimchg",
   [GTT_TYPECAST] = "typecast",
+  [GTT_ARITHMETIC] = "arithmetic",
   [GTT_END] = "error",
 };
 
+static const gchar *gst_tensor_transform_arithmetic_string[] = {
+  [ARITH_ADD] = "add",
+  [ARITH_MUL] = "mul",
+  [ARITH_END] = "error",
```

Comment:
With PROJECT:NS, I think you are going to do ```output_pixel[] = ( input_pixel[] - A ) * B```. How are you going to support it? (with two serial transform filters?)

Reply:
Well. I think add MAD mode to do that. Until we support MAD mode, we could use, transform element sequentially. add and mul. By the way, *B is not needed for PROJECT:NS.